front/src/routes/integration/all/netatmo/discover-device/DiscoverTab.jsx

```diff
@@ -0,0 +1,70 @@
+import { Component } from 'preact';
+import { Text } from 'preact-i18n';
+import cx from 'classnames';
+
+import EmptyState from '../EmptyState';
+import style from '../style.css';
+import CheckMqttPanel from '../../mqtt/commons/CheckMqttPanel';
+import TasmotaDeviceBox from '../TasmotaDeviceBox';
+
+class DiscoverTab extends Component {
+  searchDevices = () => {
+    this.props.searchDevices('mqtt');
+  };
+
+  render(props) {
+    return (
+      <div class="card">
+        <div class="card-header">
+          <h1 class="card-title">
+            <Text id="integration.tasmota.discover.mqtt.title" />
+          </h1>
+          <div class="page-options d-flex">
+            <button onClick={this.searchDevices} class="btn btn-outline-primary ml-2" disabled={props.loading}>
+              <Text id="integration.tasmota.discover.mqtt.scan" /> <i class="fe fe-radio" />
+            </button>
+          </div>
+        </div>
+        <div class="card-body">
+          <CheckMqttPanel />
+
+          <div class="alert alert-secondary">
+            <Text id="integration.tasmota.discover.mqtt.description" />
+          </div>
+          <div
+            class={cx('dimmer', {
+              active: props.loading
+            })}
+          >
+            <div class="loader" />
+            <div class={cx('dimmer-content', style.tasmotaListBody)}>
```

Comment:
Many tasmota key...

Reply:
Hi !! 
Thank you @atrovato for your review.  Well indeed all these points are not normal, it was supposed to be modified and corrected on an old merge.  I think there was a problem between our different merges. The files are fine with me and have no errors.  Sorry for the time wasted.  I correct and merge / push again tomorrow.

Reply:
Ok sorry, files deleted here : https://github.com/GladysAssistant/Gladys/pull/1015/commits/cd9f2252a960c4fbab8cb3212590c55bc8603496